Assets/scripts/LoadCatalog.cs

```diff
@@ -263,8 +263,6 @@ private void ChangeContentToCategory(string category)
     {
         GameObject content = GameObject.Find("Content");
 
-        hideListings();
-
```

Comment:
Why is this removed? This method hides all listings currently visible which should be done before you start showing other listings. This is probably why when you select a filter the old results are still visible.

Reply:
My bad. Probably did a cut-paste instead of copy-paste in the process of testing out the methods last night. I just returned it now and its working fine.